### ISSUE TITLE
no unused result false positive 

### ISSUE THREAD

**Issue by graingert** [jfmengels/eslint-plugin-lodash-fp#52]
```js
import _ from 'lodash/fp';
import assign from 'lodash';
import { someFn } from './utils';

class Foo {
    constructor() {
        assign(this, someFn());
    }
}
```

it thinks that assign is unused here, even though it mutates.
